lua/octo/pickers/telescope/entry_maker.lua

```diff
@@ -656,7 +656,8 @@ function M.gen_from_octo_actions(width)
   end
 end
 
-function M.gen_from_notification()
+function M.gen_from_notification(opts)
+  opts = opts or { show_repo = false }
```

Comment:
```suggestion
  opts = opts or { show_repo_info = false }
```
probably a slightly clearer var name for same reasons as above